libgdf/src/gdf_table.cuh

```diff
@@ -398,7 +422,7 @@ public:
   /* ----------------------------------------------------------------------------*/
   // TODO Is there a less hacky way to do this? 
   __device__
-  gdf_error get_packed_row_values(size_type row_index, byte_type * row_byte_buffer)
+  gdf_error get_packed_row_values(size_type row_index, byte_type * row_byte_buffer) const
```

Comment:
I think this function will be better if `row_byte_buffer`'s type is `void*` instead of `byte_type*`. Then inside the function you can cast `row_byte_buffer` to `byte_type*`. This prevents users of `get_packed_row_values` from needing to cast to `gdf_type::byte_type` and instead can just be passed in as a `void*`.